configure

```diff
@@ -1745,13 +1745,13 @@ Some influential environment variables:
               A description of the product which is integrating OMR. (Default:
               OMR 0.1 )
   OMR_ZOS_COMPILE_ARCHITECTURE
-              The z/OS compile machine architecture. (Default: 7)
+              The z/OS compile machine architecture. (Default: 10)
   OMR_ZOS_COMPILE_TARGET
-              The z/OS compile target operating system. (Default: zOSV1R13)
+              The z/OS compile target operating system. (Default: ZOSV2R3)
   OMR_ZOS_COMPILE_TUNE
               The z/OS compile machine architecture tuning. (Default: 10)
```

Comment:
Should we update this comment to 12 as well?

Reply:
sorry I missed this on my last update, updated now.